tests/test_cli_e2e.py

```diff
@@ -213,7 +213,7 @@ def run_step(step):
 
 
 @pytest.mark.e2e
-def test_cli_e2e(tmp_dir, catalog):
+def test_cli_e2e(tmp_dir, catalog_tmpfile):
```

Comment:
both this and test_query_e2e seem to use the parallel setting which I think means they should use the `tmp_file` variation of the catalog fixture